mica/vv/process.py

```diff
@@ -207,13 +211,15 @@ def get_arch_vv(obsid, version="last"):
         obspar_file = glob(
             os.path.join(obspar_dirs[asp_proc["obspar_version"]], "axaf*par*")
         )[0]
-    except IndexError:
-        raise LookupError(f"Requested version {version} not in obspar archive")
+    except IndexError as err:
+        raise LookupError(f"Requested version {version} not in obspar archive") from err
```

Comment:
Did you consider raise from None?

Reply:
This one I think is also acceptable as a re-labeling the error to have matching errors for the different but similar reasons that asp l1 failed lookup.

Reply:
I think there is a misunderstanding on what I mean.

I just checked this one, and it certainly should be `from None`:
```python
    except IndexError as err:
        raise LookupError(f"Requested version {version} not in obspar archive") from None
```

This does what you want, it re-labels the error. The main difference is that `from None` will not show the stack trace for the first exception. The first exception will be an `IndexError` because this 
```
glob(os.path.join(obspar_dirs[asp_proc['obspar_version']], 'axaf*par*'))
```
will be an empty list. That will cause python to print the stack trace at this point, which is redundant (it will print two stack traces at almost the same place), and the error is not informative.
```

```